notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>The <code>worker_pool_spec</code> specification consists of the following:</p><p> </p><ul><li><code>replica_count</code>: The number of VMs to provision.</li><li><code>machine_spec</code>: The hardware specification of the VMs.</li><li><code>disk_spec</code>&nbsp;: The disk storage specification, if any.</li><li><code>python_package</code>: The Python training package to install on the VM instances, the Python module to invoke, and any command line arguments for the Python module.</li></ul><p> </p><p>The <code>python_package</code> specification consists of the following:</p><p> </p><ul><li><code>executor_image_spec</code>: The docker image that is configured for your custom training job. </li><li><code>package_uris</code>: A list of the Cloud Storage bucket URIs for your Python training packages. The URIs can either point to individual Python files or an archive of an entire package. In the later case, the contents of the archive is extracted into the docker image. </li><li><code>python_module</code>: The Python module (script) to invoke for running the custom training job. In this example, you will be invoking <code>trainer.task.py</code> (appending the <code>.py</code> suffix is optional). </li><li><code>args</code>: The command line arguments to pass to the corresponding Python module. In this example, you will be setting <code>"--epochs=" + EPOCHS</code>, which is the number of epochs to train. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Changed</p>